templates/network/interfaces.d/failover-ip

```diff
@@ -0,0 +1,5 @@
+auto {{ COMMON_FAILOVER_INTERFACE_NAME }}
+iface {{ COMMON_FAILOVER_INTERFACE_NAME }} inet static
+address {{ COMMON_FAILOVER_IP }}
```

Comment:
Nit: Can you indent the last three lines by 8 spaces?  The `iface` line starts a block, and these lines belong to that block.